airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -179,3 +193,12 @@ class UserLocationReport(IncrementalGoogleAdsStream):
     UserLocationReport stream: https://developers.google.com/google-ads/api/fields/v8/user_location_view
     Google Ads API field mapping: https://developers.google.com/google-ads/api/docs/migration/mapping#geo_performance
     """
+
+
+class ClickView(IncrementalGoogleAdsStream):
+    """
+    ClickView stream: https://developers.google.com/google-ads/api/reference/rpc/v8/ClickView
+    """
+
+    time_unit = "days"
+    buffer_days = 90
```

Comment:
`buffer_days` doesn't describe this v well since a buffer is a place of aggregation. It should be something more like "oldest_allowed_data_age" (not a great name but you get the idea)